webpack.config.js

```diff
@@ -7,7 +7,36 @@ const HtmlWebpackPlugin = require("html-webpack-plugin");
 const path = require("path");
 const webpack = require("webpack");
 
-module.exports = [
+const targetList = [
+  "acorn",
+  "babel",
+  "babylon",
+  "buble",
+  "chai",
+  "coffeescript",
+  "espree",
+  "esprima",
+  "jshint",
+  "lebab",
+  "prepack",
+  "prettier",
+  "source-map",
+  "typescript",
+  "uglify-es",
+  "uglify-js"
+];
+
+function getTarget(env) {
+  return (
+    env &&
+    env.only &&
+    targetList.find(elem => {
+      return elem == env.only;
+    })
```

Comment:
What I had in mind was that `targetList` would be a `Set`. That would simplify this to `targetList.has(elem)` while still allowing easy iteration elsewhere through `for`-`of`.